java/images/fedora-java11/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM fedora:29
+
+ENV JOLOKIA_VERSION="1.6.0" \
+    PROMETHEUS_JMX_EXPORTER_VERSION="0.3.1" \
+    PATH=$PATH:"/usr/local/s2i" \
+    AB_JOLOKIA_PASSWORD_RANDOM="true" \
+    AB_JOLOKIA_AUTH_OPENSHIFT="true" \
+    JAVA_DATA_DIR="/deployments/data"
+
+# Some version information
+LABEL io.fabric8.s2i.version.maven="3.5.4" \
+      io.fabric8.s2i.version.jolokia="1.6.0" \
+      io.fabric8.s2i.version.prometheus.jmx_exporter="0.3.1" \
+      io.k8s.description="Platform for building and running plain Java applications (fat-jar and flat classpath)" \
+      io.k8s.display-name="Java Applications" \
+      io.openshift.tags="builder,java" \
+      io.openshift.s2i.scripts-url="image:///usr/local/s2i" \
+      io.openshift.s2i.destination="/tmp" \
+      org.jboss.deployments-dir="/deployments" \
+      com.redhat.deployments-dir="/deployments" \
+      com.redhat.dev-mode="JAVA_DEBUG:false" \
+      com.redhat.dev-mode.port="JAVA_DEBUG_PORT:5005"
+
+# Temporary switch to root
+USER root
+
+
+
+# Install Java package & download Maven from Apache
+RUN yum install -y \
+       java-11-openjdk \
```

Comment:
Has there been an update now which addresses that issue around huge sizes of jlink images produced with this package's java? If not, I'd not really recommend using it.

Reply:
Erm, sorry, not entirely sure I follow here ... This PR, for issue #160, just aims at providing an image with Java 11. It's a starting point. It has nothing to do, yet, with then producing custom jlink images - that would be a possible follow-up, in #181; and I was hoping you may be motivated to help with that! :smile: 

There has been no update that I know of addressing size. But perhaps you should open an issue about that point somewhere.. of course that's not really for this project here, but either (I'm not clear which) for OpenJDK itself (so on https://openjdk.java.net), or for the OpenJDK package in Fedora/CentOS - do you want to try opening something on https://bugzilla.redhat.com ? Do link it on #181 when you do.

Or are you really arguing here that the java-11-openjdk YUM package as is cannot be currently used? Why? Because it's too big? Is that really a blocker for even getting started with something to let folks explore Java 11?

Reply:
> Or are you really arguing here that the java-11-openjdk YUM package as is cannot be currently used?

There's no issue with the size of the package itself, only any jlink images produced via this package will be very large. Hence I'd personally not use that package for the time being and instead get the tar.gz from openjdk.net.

Reply:
Ok, I've asked internally where/how to report this issue.

@vorburger, in doing so, I realized that for using jlink another package must be installed: java-11-openjdk-jmods. This brings in the jmod files with the JDK modules that are used by jlink to produce the modular runtime images.

Reply:
see #198 for #181